src/combine.cpp

```diff
@@ -1041,6 +1041,17 @@ void do_symbol_check(chunk_t *prev, chunk_t *pc, chunk_t *next)
       if (chunk_is_paren_open(tmp))
       {
          set_paren_parent(tmp, CT_DECLTYPE);
+
+         // decltype may be followed by a braced-init-list
+         tmp = chunk_get_next_type(tmp, CT_PAREN_CLOSE, tmp->level);
```

Comment:
Instead of this and calling `chunk_get_next_ncnl`, please instead use the return value of `set_paren_parent` (previous line), which is the same token, but avoids doing the searches twice.